src/main/java/com/dsmbamboo/api/domains/users/security/SecurityConfiguration.java

```diff
@@ -32,7 +32,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionManagement()
                     .sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
                 .authorizeRequests()
-                    .antMatchers("/auth/signin").permitAll()
+                    .antMatchers("/auth/admin").permitAll()
+                    .antMatchers("/auth/anonymous").permitAll()
```

Comment:
```suggestion
                    .antMatchers("/admin/auth/signin").permitAll()
                    .antMatchers("/auth/signin").permitAll()
```
요게 더 적절할 것 같아요

rest 디자인 관련 좋은 글: https://bcho.tistory.com/914
요거 보고 api 버저닝도 경험해보면 좋을거 같아요